game/scripts/npc/abilities/talents/storm_spirit_talent6.txt

```diff
@@ -18,7 +18,7 @@
       "01"
       {
         "var_type"        "FIELD_INTEGER"
-        "value"           "250" //OAA
+        "value"           "125" //OAA
```

Comment:
Sadge

Reply:
200% of the level 4 damage as a talent is kinda yikes